test/session.js

```diff
@@ -2294,6 +2304,37 @@ describe('session()', function(){
       })
     })
   })
+
+  describeHttp2('http2', function () {
+    it('should work with http2 server', function (done) {
+      var server = createHttp2Server(null, function (req, res) {
+        res.setHeader(http2.constants.HTTP2_HEADER_CONTENT_TYPE, 'text/plain')
+        res.end('Hello, world!')
+      })
+      server.on('listening', function () {
+        var client = createHttp2Client(server.address().port)
+        // using ES5 as Node.js <=4.0.0 does not have Computed Property Names
+        var reqHeaders = {}
+        reqHeaders[http2.constants.HTTP2_HEADER_PATH] = '/'
```

Comment:
It doesn't seem this is necessary, as https://nodejs.org/dist/latest-v18.x/docs/api/http2.html#clienthttp2sessionrequestheaders-options says that `/` is the default path (just like `GET` is the default method, and you're not specifying the method here).

Reply:
Good catch. Will remove it for consistency and less verbosity,